### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by VyacheslavKokayko**
- [DEMO LINK](https://VyacheslavKokayko.github.io/layout_catalog/)
- [TEST REPORT LINK](https://VyacheslavKokayko.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,203 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
```
Comment (VitaliyBondarenko1982) [inline src/styles/main.scss:4]: redundant reset margin and padding for all selectors. reset them only for elements which need it
```suggestion
```

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,203 @@
...
+}
+
+html,
+body {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
```
Comment (VitaliyBondarenko1982) [inline src/styles/main.scss:10]:
```suggestion
html {
  font-family: Roboto, sans-serif;
}
```
enough to set for html.
looks like somethin went wrong with font. try to set font-family such as in suggestion

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,203 @@
...
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  height: 60px;
+  display: flex;
+  align-items: center;
```
Comment (VitaliyBondarenko1982) [inline src/styles/main.scss:46]:
```suggestion
```
looks like redundant styles.

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,203 @@
...
+    margin-right: 20px;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.nav__link {
+  height: 60px;
```
Comment (VitaliyBondarenko1982) [inline src/styles/main.scss:58]:
```suggestion
  padding: 23px 0;
```
set directly height is bad practice. instead make vertical paddings

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,203 @@
...
+  text-transform: uppercase;
+  color: #060b37;
+  transition: color 0.3s;
+
+  &:hover {
+    color: #00acdc;
+  }
+
+  &.is-active {
+    color: #00acdc;
```
Comment (VitaliyBondarenko1982) [inline src/styles/main.scss:74]: make variables for colors which uses more than once

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,203 @@
...
+      }
+    }
+
+    &--1 .stars__star:nth-child(-n + 1),
+    &--2 .stars__star:nth-child(-n + 2),
+    &--3 .stars__star:nth-child(-n + 3),
+    &--4 .stars__star:nth-child(-n + 4),
+    &--5 .stars__star:nth-child(-n + 5) {
+      background-image: url(../images/star-active.svg);
+    }
```
Comment (VitaliyBondarenko1982) [inline src/styles/main.scss:201]: pay attention on checklist [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)
use sass loop for stars block
Screenshot from 2023-10-07 15-35-07

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Good job.
Needs some improvements.
Please check comments and tru to fix tests.
Also make separate file for each BEM block (header, catalog, card, stars ...) and import all files in main.scss

File: src/index.html [hunk 7]
```diff
@@ -4,9 +4,370 @@
...
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+    
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+    
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:56]: Let's make links more informative
```suggestion
            <a href="#photo" class="nav__link">Photo</a>
```

File: src/styles/main.scss (hunk 2 above)
Comment (volodymyr-soltys97) [reply src/styles/main.scss:10]: Still not fixed from the previous review

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: backstop_data/engine_scripts/puppet/onReady.js
Comment (loralevitska) [inline backstop_data/engine_scripts/puppet/onReady.js:1]: In the future - Do not add extra files to PR. To add the file to a commit use `git add src/.`

File: src/index.html (hunk 7 above)
Comment (loralevitska) [inline src/index.html:56]: Not fixed. Fix all cases
image

File: src/styles/main.scss (hunk 1 above)
Comment (loralevitska) [inline src/styles/main.scss:4]: Not fixed 
image

File: src/styles/main.scss [hunk 8]
```diff
@@ -1,3 +1,203 @@
...
+  column-gap: 48px;
+  justify-content: center;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
```
Comment (loralevitska) [inline src/styles/main.scss:26]: It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

**Review by loralevitska — CHANGES_REQUESTED** [review]
Hi, I see you haven't fixed previous comments, maybe you did not push your changes? If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

Please redeploy your solution and check links before re-request review
image

**Pull request closed without merging**